packages/inventory/src/components/table/EntityTableToolbar.js

```diff
@@ -177,31 +177,31 @@ const EntityTableToolbar = ({
     };
 
     useEffect(() => {
-        if (page && perPage && filters && showTags && !hasItems) {
+        if (page && perPage && filters && showTags && !hasItems && loaded) {
```

Comment:
What about to compute this calue before the hooks and then use the constant to deduplicate and clean this code?

```jsx
const shouldReload = page && perPage && filters && (!hasItems || items) && loaded;

useEffect(() => {
     shouldReload && onSetFilter(...)
}, ...)

useEffect(() => {
     shouldReload && onSetTextFilter(...)
}, ...)

...
```

Reply:
Good point! I'd still use an `if` statement instead of shortcutting so anybody reading this later on would have easier to understand what that means.